src/components/NewNursery/index.tsx

```diff
@@ -0,0 +1,111 @@
+import { Container, Grid, Typography, useTheme } from '@mui/material';
+import { useState } from 'react';
+import { useHistory } from 'react-router-dom';
+import { APP_PATHS } from 'src/constants';
+import strings from 'src/strings';
+import { ServerOrganization } from 'src/types/Organization';
+import TextField from '../common/Textfield/Textfield';
+import useForm from 'src/utils/useForm';
+import FormBottomBar from '../common/FormBottomBar';
+import { Facility } from 'src/api/types/facilities';
+import { createFacility } from 'src/api/facility/facility';
+import useDeviceInfo from 'src/utils/useDeviceInfo';
+import PageSnackbar from 'src/components/PageSnackbar';
+import useSnackbar from 'src/utils/useSnackbar';
+
+type SiteViewProps = {
+  organization: ServerOrganization;
+  reloadOrganizationData: () => void;
+};
+
+export default function NurseryView({ organization, reloadOrganizationData }: SiteViewProps): JSX.Element {
+  const [nameError, setNameError] = useState('');
+  const [descriptionError, setDescriptionError] = useState('');
+  const snackbar = useSnackbar();
+  const theme = useTheme();
+
+  const [record, , onChange] = useForm<Facility>({
+    name: '',
+    id: -1,
+    type: 'Nursery',
+    organizationId: organization.id,
+    connectionState: 'Not Connected',
```

Comment:
remove?

Reply:
connectionState is required, maybe I should ask Steve to make it optional ?